docs/src/pages/getting-started/installation/dependencies.react.mdx

```diff
@@ -51,3 +55,42 @@ export default function App(props) {
 ```
 </TabItem>
 </Tabs>
+
+> The `AmplifyProvider` helps direct [theming](/theming) to all the Amplify components in your application. It can optionally be used with a [theme object](/theming#theme-object), to set the look and feel of your app.
+
+### Authenticator Provider
+
+If you're planning on using the [Authenticator](/components/authenticator) with the [useAuthenticator hook](/components/authenticator#useauthenticator-hook), for advanced use cases, wrap your application in the `Authenticator.Provider` component.
+
+<Tabs>
+<TabItem title="Create React App">
+```jsx
+import { AmplifyProvider, Authenticator } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css'; // default theme
+
+export default function App() {
+  return (
+    <Authenticator.Provider>
+      <App />
+    </Authenticator.Provider>
+  );
+};
+```
+</TabItem>
+<TabItem title="Next.js" >
+```jsx
+import { AmplifyProvider, Authenticator } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css'; // default theme
+
+export default function App(props) {
+  return (
+    <Authenticator.Provider>
+      <MyApp {...props} />
```

Comment:
Would be nice if this could copy paste into customer apps without any errors from missing `MyApp` component.
```suggestion
      <View>
       Your app here
      </View>
```

Reply:
Done